core/lib/dal/src/transactions_dal.rs

```diff
@@ -42,6 +43,7 @@ impl fmt::Display for L2TxSubmissionResult {
             Self::AlreadyExecuted => "already_executed",
             Self::Duplicate => "duplicate",
             Self::Proxied => "proxied",
+            Self::NonceInProgress => "nonce_in_progress",
```

Comment:
oh @perekopskiy meant to approve, not request changes.

Reply:
I didn't use replacement because it actually can be not a replacement but insertion but I think it's ok to sacrifice it and call it replacement for simplicity

Reply:
ah good point.. `InsertionInProgress`? hehe